src/js/utils/__tests__/mixins-test.js

```diff
@@ -0,0 +1,36 @@
+import { parseMetricToNum } from '../mixins';
+
+afterAll(() => {
+  jest.clearAllMocks();
+});
+
+describe('Mixins parseMetricToNum', () => {
+  global.console = {
+    warn: jest.fn(),
+  };
+
+  test('coverts "12px" to 12', () => {
+    const number = parseMetricToNum('12px');
+
+    expect(number).toBe(12);
+  });
+
+  test('coverts "12.5px" to 12', () => {
+    const number = parseMetricToNum('12.5px');
+
+    expect(number).toBe(12.5);
+  });
+
+  test('coverts "12px 20px" to 12', () => {
+    const number = parseMetricToNum('12px 20px');
+
+    expect(number).toBe(12);
+  });
+
+  test('warns when caller expects "12px 20px" in a theme override', () => {
+    const warn = jest.spyOn(global.console, 'warn');
+    parseMetricToNum('12px 20px');
+
```

Comment:
we can merge this test and the previous one no?

Reply:
Yes - modified. I have the same input and 2 expect statements now. Also improved with `expect(warn).toHaveBeenCalledWith('This feature takes only 1 value and should have no spaces');` to check if the message part is alright.